DEVELOPMENT.md

```diff
@@ -87,6 +87,27 @@ Maintain the same quality for production and test code.
 
 When writing a Git commit message, follow these [guidelines](https://chris.beams.io/posts/git-commit/).
 
+### Git merge strategy
+
+Pull requests are usually merged into `master` using the  [`rebase and merge`](https://docs.github.com/en/github/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/about-pull-request-merges#rebase-and-merge-your-pull-request-commits) strategy.
+
+A typical pull request should strive to just contain one commit - logically unrelated changes should generally be extracted
+into their own PRs.
+
+If a pull request does consist of multiple commits, it is expected that every prefix of it is correct. That is,
+there might be preparatory commits at the bottom of the stack that don't bring any value by themselves,
+but none of the commits should introduce an error that is fixed by some future commit.
+
+It is important to keep commits on feature branches neat, squashing the feature branch as necessary.
+
+Commit messages and  history are also important, as they are used by other developers to keep track of
```

Comment:
double space

Reply:
Fixed - also, checked the file for double spaces and there are a lot of double spaces after periods. Is that the correct convention?

Reply:
We should remove these. Can you please fix them in separate commit?